### ISSUE TITLE
[Question] why does CLS attribution identify the first element, not the one with the largest shift?

### ISSUE THREAD

**Issue by williazz** [GoogleChrome/web-vitals#375]
Hi I have a general question. My goal is to identify the elements responsible for the largest shift contributing to the CLS score. Why does the CLS Attribution identify the first element, not the one with the largest shift? 

Element with largest shift seems useful, since site owners want to identify individual elements that are likely to be accidentally clicked. And the first element may not even be a large contributor to the worst shifts. What is the diagnostic value of identifying the first element?

**Comment by tunetheweb** [comment]
It was discussed here: https://github.com/GoogleChrome/web-vitals/issues/309#issuecomment-1399410173

TLDR: as we identify the shift*ed* elements and not the ones that cause the shift, the first shifted element often is closer to the cause.

**Comment by williazz** [comment]
Thanks for pointing me to that thread.
